examples/c/profile.bpf.c

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2022 Meta Platforms, Inc. */
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+#include "profile.h"
+
+char LICENSE[] SEC("license") = "Dual BSD/GPL";
+
+struct {
+	__uint(type, BPF_MAP_TYPE_RINGBUF);
+	__uint(max_entries, sizeof(struct stacktrace_event) * 64);
```

Comment:
with most recent libbpf this would work because libbpf will auto-adjust the size to the closest multiple of page size, but I think it's a bit confusing to express the size like this. Let's just have something like `256 * 1024` specified as a size?

Reply:
Ok!